workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -32,6 +33,9 @@
             "physics_precip",
             "column_integrated_dQu",
             "column_integrated_dQv",
+            "cnvprcp_after_python",
+            "area",
+            "total_precipitation_rate",
```

Comment:
Does this list actually do anything then since it's called `output_variables` instead of `variables`? Visually I assumed it was doing what it would do if it were called `variables`. If that's actually the case then please delete this list.

Reply:
Good point. I deleted it.